src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
@tannerlinsley That means it will be accessible on the external dimension API. I'd really prefer to keep the external API minimal. I think the original CF team did a really good job of that.

@drarmstr You're right - I had a brain malfunction :-) You'd need to maintain an array of dimension object references, and corresponding arrays of dimension offsets, etc. Then do an indexOf lookup on the dimension array and grab the corresponding index from the offset array. You'd want to maintain the arrays (splice) when dimension.dispose is called. If you want, I can do this after accepting the pull request onto a branch. Just let me know.